app/models/discussion_lock.rb

```diff
@@ -0,0 +1,7 @@
+class DiscussionLock < ApplicationRecord
+  belongs_to :article
+  belongs_to :user
```

Comment:
It's not always worth the effort but when defining associations on common relations like `User` I often like to use `class_name` to give the relation a better, more context-specific name, something like:

```ruby
belongs_to :locking_user, class_name: 'User'
```

This model is probably small and clear enough to not need this but I do think it's something to keep in mind as it can really disambiguate code for other developers.